mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageIdMapper.java

```diff
@@ -289,7 +289,7 @@ public void delete(MessageId messageId) {
         return imapUidDAO.updateMetadata(newComposedId, oldComposedId.getModSeq())
             .thenCompose(updateSuccess -> Optional.of(updateSuccess)
                 .filter(b -> b)
-                .map(any -> messageIdDAO.updateMetadata(newComposedId).thenApply(v -> updateSuccess))
+                .map((Boolean any) -> messageIdDAO.updateMetadata(newComposedId).thenApply(v -> updateSuccess))
```

Comment:
Which one is going to be merge first ;-) https://github.com/linagora/james-project/pull/575/commits/658d5cfcae3362b1de754f82c863bb5c30025860